sdk/smx.ts

```diff
@@ -36,6 +40,47 @@ class SMXEvents {
       .withStateMachine({ currentPacket: new Uint8Array() }, handlePacket);
 
     // this.otherReports$.onValue((value) => console.log("Packet: ", value));
+
+    this.otherReports$
+    .filter((e) => e.type === 'host_cmd_finished')
+    .onValue((e) => console.log("Cmd Finished"));
+
+    const finishedCommand$ = this.otherReports$
+      .filter((e) => e.type === 'host_cmd_finished')
+      .map((e) => e.type === 'host_cmd_finished');
+
+    this.startedSend$ = new Bacon.Bus<boolean>();
+
+    // true means "it's ok to send", false means "don't send"
+    const okSend$ = new Bacon.Bus<boolean>()
+      .merge(finishedCommand$)  // Returns true when host_cmd_finished
+      .merge(this.startedSend$.not())  // Return false when starting to send
+      .toProperty(true);
```

Comment:
so I think theoretically this works...

Reply:
does this even need to be a bus in this case? we don't need to write to it at all, so I think we just start with either of the merged streams, merge the other to it, and convert to a property

Reply:
oh yeah good point, that probably makes more sense.